src/GraphQL/Types/QueryArgument.cs

```diff
@@ -92,7 +92,7 @@ public IGraphType ResolvedType
         public Type Type
         {
             get => _type;
-            private set => _type = CheckType(value);
+            internal set => _type = CheckType(value);
```

Comment:
Because this is not `public`, and because `SchemaTypes` does not expose any `protected` members that can be overridden to patch the issue, there is no way for users to write a simple patch to fix the bug.